prebuilt_forms/species_details_2.php

```diff
@@ -984,6 +1086,9 @@ protected static function get_control_recsthroughyear($auth, $args, $tabalias, $
    *   The output map.
    */
   protected static function get_control_exploremap($auth, $args, $tabalias, $options) {
+    if (self::$notaxon) {
```

Comment:
I'm not sure if it's easy to change, but could the form just not get built at all if notaxon is true, rather than building it still but returning '' for all the controls?

Reply:
If the [species] - taxon selector - control is specified on the form config, then the form needs to get built whether or not a taxon is specified in the URL which is why I took that approach.